client/client.go

```diff
@@ -10,23 +10,27 @@ import (
 
 // Binder for the Hyperdrive daemon API server
 type ApiClient struct {
-	context client.IRequesterContext
-	Service *ServiceRequester
-	Tx      *TxRequester
-	Utils   *UtilsRequester
-	Wallet  *WalletRequester
+	context           client.IRequesterContext
+	NodeSet           *NodeSetRequester
+	NodeSet_StakeWise *NodeSetStakeWiseRequester
```

Comment:
NodeSetStakeWise for consistency?

Reply:
I added an underscore mainly to distinguish that `_StakeWise` is actually a subrote of `NodeSet` (i.e. the route is `/api/v1/nodeset/stakewise`) - I'd expect to do that with any client requester that involves a subroute.